spec/models/knight_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Knight, type: :model do
+  describe '#create' do
+    it 'creates a knight in the database' do
+      knight = FactoryGirl.create(:knight)
+      expect(knight.type == 'Knight').to eq(true)
+    end
+  end
+

```

Comment:
I think we'd want to write some tests to try out that obstruction logic override here